build.sbt

```diff
@@ -82,3 +82,13 @@ lazy val controllers =
   (project in file("controllers"))
     .settings(libraryDependencies ++= akkaDependencies ++ jsonDependencies :+ cats)
     .dependsOn(services, utils, repositories % "test->test")
+
+lazy val womtool = (project in file("womtool"))
+  .configs(IntegrationTest)
+  .settings(
+    resolvers += Resolver.bintrayRepo("scalalab", "pipeline"),
+    name := "WomTool",
+    commonSettings,
+    libraryDependencies ++= testContainers ++ testDependencies ++ cromwellDependencies,
```

Comment:
Do we really need `testContainers` dependency in this module?

Reply:
resolved